src/main/java/at/ac/fhcampuswien/fhmdb/HomeController.java

```diff
@@ -3,10 +3,9 @@
 import at.ac.fhcampuswien.fhmdb.database.DatabaseException;
 import at.ac.fhcampuswien.fhmdb.database.WatchlistRepository;
 import at.ac.fhcampuswien.fhmdb.interfaces.ClickEventHandler;
-import at.ac.fhcampuswien.fhmdb.models.Genre;
-import at.ac.fhcampuswien.fhmdb.models.Movie;
-import at.ac.fhcampuswien.fhmdb.models.MovieAPI;
-import at.ac.fhcampuswien.fhmdb.models.MovieApiException;
+import at.ac.fhcampuswien.fhmdb.models.*;
+import at.ac.fhcampuswien.fhmdb.states.AscendingState;
```

Comment:
```suggestion
import at.ac.fhcampuswien.fhmdb.states.*;
```

Use wildcard for states (like models)